relay-pii/src/convert.rs

```diff
@@ -1656,4 +1656,45 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         process_value(&mut data, &mut pii_processor, ProcessingState::root()).unwrap();
         assert_annotated_snapshot!(data);
     }
+
+    #[test]
+    fn test_pairlist_scrubbed() {
+        let mut data = Event::from_value(
+            serde_json::json!({
+                "threads": {
+                    "values": [
+                        {
+                            "stacktrace": {
+                                "frames": [
+                                    {
+                                        "vars": {
+                                            "request": {
+                                                "headers": [
+                                                    [
+                                                        "secret",
+                                                        "A1BBC234QWERTY0987MNBV012765HJKL"
+                                                    ],
+                                                    [
+                                                        "passwd",
+                                                        "my_password"
+                                                    ]
```

Comment:
Can we split this up to test both the key-detection and the value detection?

```suggestion
                                                    [
                                                        "passwd",
                                                        "asdf123"
                                                    ],
                                                    [
                                                        "something_else",
                                                        "my_password"
                                                    ]
```